addons/api/addon/handlers/client-daemon-handler.js

```diff
@@ -75,14 +76,48 @@ const ClientDaemonHandler = {
           'query',
         );
 
-        // TODO: Add some pagination
+        // Return the raw data if we don't push to the store.
+        let records = normalizedPayload.data;
```

Comment:
This unfortunately means we wouldn't be returning models, do we feel OK with that?